css/stylesheet.css

```diff
@@ -0,0 +1,88 @@
+header {
+	background-image: url(../about_me/misty81.jpg);
+	background-size: cover;
+	height: 500px;
+	color: white;
+	font-family: sans-serif;
+	text-align: left;
+	font-size: 100px
+	color: white;
+	text-transform: uppercase;
+	padding: 40px 0px 0px 40px;
+	margin: 40px;
+	line-height: 1.25;
+	text-shadow: 2px 2px 5px black;
+}
+
+body {
```

Comment:
This rule should be first so that the rest of the stylesheet can inherit from it. :hammer: